.github/workflows/reusable_sign-artifacts.yaml

```diff
@@ -58,8 +59,15 @@ jobs:
   sign:
     runs-on: ${{ inputs.runs-on }}
     env:
-      CHECKOUT_REF: 722626ac96e0e8569ba587bd228b9468f6e49ead # Latest commit hash for development
+      CHECKOUT_REF: c75407f2cd34683f69f302aaf8fa425a5623ea11 # Latest commit hash for development
     steps:
+      - name: Download Unsigned Artifacts
+        uses: actions/download-artifact@v4
+        with:
+          pattern: ${{ inputs.unsigned-artifact-pattern }}
```

Comment:
Should this match the pattern in the execute-build workflow runs?

https://github.com/aerospike/shared-workflows/blob/080edddc84f49bf1a049cebe9943e53052dcde71/.github/workflows/reusable_sign-artifacts.yaml#L95-L98

Reply:
I think you are right that it would be nicer if the defaults chained nicely.